pom.xml

```diff
@@ -41,6 +41,22 @@
         <docker.showLogs>true</docker.showLogs>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>oss</id>
+            <name>OSS Sonatype Nexus</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>never</updatePolicy>
+            </snapshots>
+        </repository>
+    </repositories>
```

Comment:
@jpechane, unfortunately I already merged two of the PRs, but I think this is going to be a problem. AFAIK, no POM deployed to Maven Central may contain repository information.

Reply:
@gunnarmorling Interesting, we have it in Debezium parent pom for ages - since I started on the project

Reply:
Hum, that's interesting indeed now. So I guess I'll just merge all of them then?